panic: no Auth Provider found for name "gcp"

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
Copied kubectl-wait-sts into classpath
Run:
kubectl wait-sts foo -n bar

Thown:
panic: no Auth Provider found for name "gcp"

goroutine 1 [running]:
k8s.io/client-go/kubernetes/typed/admissionregistration/v1.NewForConfigOrDie(...)
	/home/runner/go/pkg/mod/k8s.io/client-go@v0.18.1/kubernetes/typed/admissionregistration/v1/admissionregistration_client.go:64
k8s.io/client-go/kubernetes.NewForConfigOrDie(0xc0001e0200, 0xc0001e0200)
	/home/runner/go/pkg/mod/k8s.io/client-go@v0.18.1/kubernetes/clientset.go:552 +0x11f3
github.com/oke-py/kubectl-wait-sts-plugin/pkg/cmd.(*WaitStsOptions).Run(0xc00003aa80, 0x0, 0x0)
	/home/runner/work/kubectl-wait-sts-plugin/kubectl-wait-sts-plugin/pkg/cmd/wait-sts.go:101 +0xe2
github.com/oke-py/kubectl-wait-sts-plugin/pkg/cmd.NewCmdWaitSts.func1(0xc0000c6b00, 0xc0003cc030, 0x1, 0x3, 0x0, 0x0)
	/home/runner/work/kubectl-wait-sts-plugin/kubectl-wait-sts-plugin/pkg/cmd/wait-sts.go:47 +0x74
github.com/spf13/cobra.(*Command).execute(0xc0000c6b00, 0xc00003a110, 0x3, 0x3, 0xc0000c6b00, 0xc00003a110)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc0000c6b00, 0x0, 0x1273120, 0xc00009a058)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
main.main()
	/home/runner/work/kubectl-wait-sts-plugin/kubectl-wait-sts-plugin/cmd/kubectl-wait_sts.go:10 +0x27


**What you expected to happen**:
Show it worked ok 

**Anything else we need to know?**:
kubectl version
Client Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.0", GitCommit:"e8462b5b5dc2584fdcd18e6bcfe9f1e4d970a529", GitTreeState:"clean", BuildDate:"2019-06-19T16:40:16Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14+", GitVersion:"v1.14.10-gke.27", GitCommit:"145f9e21a4515947d6fb10819e5a336aff1b6959", GitTreeState:"clean", BuildDate:"2020-02-21T18:01:40Z", GoVersion:"go1.12.12b4", Compiler:"gc", Platform:"linux/amd64"}

I have more than one cluster and project in kubectl config.

--- a/cmd/kubectl-wait_sts.go
+++ b/cmd/kubectl-wait_sts.go
@@ -1,8 +1,10 @@
 package main
 
 import (
-	"github.com/oke-py/kubectl-wait-sts-plugin/pkg/cmd"
 	"os"
+
+	"github.com/oke-py/kubectl-wait-sts-plugin/pkg/cmd"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 )
 
 func main() {

